Makefile

```
@@ -19,12 +19,12 @@ ci-like-lint :
 lint :
 	golangci-lint run --out-format=tab --new-from-rev master ./...
 
-tests :
+test :
```

```suggestion
test:
```
Is the space expected? I see we have it on every command

---

:+1: